.github/workflows/project.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
     name: Tests
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
?? what the reason to use mac? they are usually more expensive than other machines to run your builds